src/DIRAC/FrameworkSystem/Service/TornadoProxyManagerHandler.py

```diff
@@ -0,0 +1,14 @@
+""" ProxyManager is the implementation of the ProxyManagement service in the DISET framework
+
+    .. literalinclude:: ../ConfigTemplate.cfg
+      :start-after: ##BEGIN ProxyManager:
+      :end-before: ##END
+      :dedent: 2
+      :caption: ProxyManager options
+"""
+from DIRAC.Core.Tornado.Server.TornadoService import TornadoService
+from DIRAC.FrameworkSystem.Service.ProxyManagerHandler import ProxyManagerHandlerMixin
+
+
+class ProxyManagerHandler(ProxyManagerHandlerMixin, TornadoService):
```

Comment:
This should be called 
```suggestion
class TornadoProxyManagerHandler(ProxyManagerHandlerMixin, TornadoService):
```